### PR TITLE
Delaynie/shooter tuning

### PR DESCRIPTION
**Pull request by thetaspirit**

### CODE REVIEW

**Comment by thetaspirit** [comment]
still need to chain method for setting the timeout on the ShootOneBallCommand

File: code/src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 1]
```diff
@@ -22,111 +24,155 @@
...
+        int setpoint = Preferences.getInstance().getInt("shooterTargetRPM", shooterRPM);
+        m_shooterPID.setSetpoint(setpoint);
+        SmartDashboard.putNumber("ShooterTargetRPM", setpoint);
     }
 
+    /**
+     * basically, this makes sure one ball is yeeted, if feedbackward is false. if
+     * feedbackward is true, it just drives the feeder in the opposite direction,
+     * probably becasue mech didn't fix their problems, and the balls got jammed
+     */
```
Comment (noam987) [inline code/src/main/java/frc/robot/subsystems/ShooterSubsystem.java:95]: what is this corresponding to?

**Review by noam987 — DISMISSED** [review]

**Review by noam987 — APPROVED** [review]

File: code/src/main/java/frc/robot/subsystems/ShooterSubsystem.java (hunk 1 above)
Comment (thetaspirit) [reply code/src/main/java/frc/robot/subsystems/ShooterSubsystem.java:95]: that boolean shouldnt be there why is it there??

Comment (thetaspirit) [reply code/src/main/java/frc/robot/subsystems/ShooterSubsystem.java:95]: it's fine for now.  if it works, just leave it

**Pull request merged**